README.md

```diff
@@ -178,6 +178,71 @@ excludes all algorithms of the "Sphincs" family.
 *Note*: By default, interoperability testing with oqs-openssl111 is no longer
 performed by default but can be manually enabled in the script `scripts/runtests.sh`.
 
+Building under Windows
+--------------------
+The instructions below were tested on Windows 10 and 11 using MSYS2 MINGW64 (building with Visual Studio 2019 was NOT successful in our case). They are a bit experimental and only provisional. Note that the tests were skipped in the building process, however, setting up a test server and client with PQC algorithms still works.
```

Comment:
A statement like "in our case" reads weird if part of the project README. Could you please rephrase to state "...was not successful and still is an [open work item](https://github.com/open-quantum-safe/oqs-provider/issues/47]"? Also, any lessons learned (what went wrong, what should be looked into to get MSVC to work) would be a great addition to the comments section in https://github.com/open-quantum-safe/oqs-provider/issues/47.
Also regarding testing: If this doesn't work, can you state why? Is it something that can be fixed easily (.bat file instead of .sh, for example) or that is so complex that the platform should be excluded from testing completely, e.g., in https://github.com/open-quantum-safe/oqs-provider/blob/main/test/CMakeLists.txt?